docs/developers/building-open-mpi.rst

```diff
@@ -1,9 +1,87 @@
 Building Open MPI
 =================
 
+General
+-------
+
 Once you have run ``autogen.pl`` successfully, you can configure and
 build Open MPI just like end users do with official distribution Open
 MPI tarballs.
 
 See the :doc:`general "Install Open MPI" documentation for more
 details. </installing-open-mpi/index>`
+
+Building Against External OpenPMIx / PRRTE
+------------------------------------------
+
```

Comment:
I was under the impression that you were going to remove the ability to use an external PRRTE - no longer true? People can, of course, always use an independent version anyway, but this really referenced the ability to "link" `mpirun` to execute `prte` from the external PRRTE installation.

Reply:
Probably true.  But that's in the future -- this document is for `main` and `v5.0.x`, where you can still use an external PRTE.